config/locales/telephony/en.yml

```diff
@@ -3,24 +3,24 @@ en:
   telephony:
     account_reset_cancellation_notice: Your request to delete your %{app_name} account has been cancelled.
     account_reset_notice: As requested, your %{app_name} account will be deleted in
-      24 hours. Don't want to delete your account? Sign in to your %{app_name}
+      24 hours. Don’t want to delete your account? Sign in to your %{app_name}
       account to cancel.
     authentication_otp:
       sms: |-
-        %{app_name}: Your security code is %{code}. It expires in %{expiration} minutes. Don't share this code with anyone.
+        %{app_name}: Your security code is %{code}. It expires in %{expiration} minutes. Don’t share this code with anyone.
```

Comment:
looks like the spec hardcoded the translation for this one with the straight quote, so that's why the build is failing

Reply:
> looks like the spec hardcoded the translation for this one with the straight quote, so that's why the build is failing

Thanks 👍  Should be fixed in 3bffbb9.